core/server/data/importer/importers/data/posts.js

```diff
@@ -99,10 +104,37 @@ class PostsImporter extends BaseImporter {
 
     beforeImport() {
         debug('beforeImport');
+        let mobileDocContent, self = this;
 
         this.sanitizeAttributes();
         this.addTagsToPosts();
 
+        self.dataToImport = this.dataToImport.map(self.legacyMapper);
+
+        // For legacy imports/custom imports with only html we can parse the markdown or html into a mobile doc card
+        // For now we can hardcode the version
+        _.each(self.dataToImport, function (model) {
+            if (_.isNull(model.mobiledoc)) {
```

Comment:
`isNull` must change to `!model.mobiledoc`, because it can be either null or undefined?